build.gradle

```diff
@@ -561,3 +561,27 @@ gradle.projectsEvaluated {
     }
   }
 }
+
+apply plugin: 'compare-gradle-builds'
```

Comment:
I wonder if we should keep this long term. I just feel like we won't need it again for six months and when we want it again we'll have to edit all of this to make it work.

Reply:
Is it worth applying the plugin at all if we don't have `build.compare` set?